src/pages/home/report/ReportActionItemFragment.js

```diff
@@ -152,8 +160,15 @@ function ReportActionItemFragment(props) {
                 </Text>
             );
         }
-        case 'TEXT':
-            return (
+        case 'TEXT': {
+            return props.isApprovedOrSubmittedReportAction ? (
+                <Text
+                    numberOfLines={props.isSingleLine ? 1 : undefined}
+                    style={[styles.chatItemMessage, styles.colorMuted]}
```

Comment:
I think we want `styles.ltr` here too, like for the `COMMENT` case above.

Reply:
Oops reading things out of order, ignore me! :sweat_smile:

Reply:
For what it's worth, adding `styles.ltr` there made no difference. 

Applying it to the wrapper `Text` in `ReportActionItemMessage` was enough.